inspirehep/modules/orcid/utils.py

```diff
@@ -236,53 +78,39 @@ def _get_api_url_for_recid(server_name, api_endpoint, recid):
     return urljoin(api_url, recid)
 
 
-def _get_record_by_mime(config, recid, mime_type):
-    """
+def get_orcid_recid_key(orcid, rec_id):
+    """Return the string 'orcid:``orcid_value``:``rec_id``'"""
+    return 'orcid:{}:{}'.format(orcid, rec_id)
```

Comment:
The value in this key is a putcode, thus I would call this something like: `'orcidputcode:{}:{}'.format(orcid, rec_id)`